scripts/scripts.js

```diff
@@ -617,7 +617,7 @@ export function loadBlogData() {
   fetch(`/query-index.json?offset=${offset}`)
     .then((response) => response.json())
     .then((responseJson) => {
-      window.blogindex.data = responseJson?.data?.filter((entry) => entry.path.startsWith('/blog/')) || [];
+      window.blogindex.data = responseJson?.data?.filter((entry) => entry.path.startsWith('/blog/') && entry.path !== '/blog/archive') || [];
```

Comment:
I prefer multiple chained filters over a big boolean expression here, as this will make it easier to understand for the next coding agent to touch this code.